.github/workflows/deploy-docs.yml

```diff
@@ -47,4 +47,27 @@ jobs:
         run: |
           make html SPHINXOPTS=-W
           tools/deploy_documentation.sh
-        shell: bash
\ No newline at end of file
+        shell: bash
+  deploy-translatable-strings:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -U virtualenv setuptools wheel tox
+        pip install jupyter sphinx_rtd_theme 'matplotlib<3.3.0' qiskit-terra[visualization] torchvision
```

Comment:
Simply copied these dependencies from Qiskit/qiskit and L38

Reply:
Fixed in https://github.com/Qiskit/qiskit-machine-learning/pull/130/commits/ac9475edfc896091d6166818a1fda47e897951ee